app/src/main/java/io/github/sds100/keymapper/sorting/ObserveKeyMapsSorterUseCase.kt

```diff
@@ -0,0 +1,40 @@
+package io.github.sds100.keymapper.sorting
+
+import io.github.sds100.keymapper.mappings.keymaps.KeyMap
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+
+/**
+ * Observes the key maps comparator based on the user's preferences.
+ */
+class ObserveKeyMapsSorterUseCase(
```

Comment:
Can we merge all the sorting use cases into one use case? Just call it `SortKeyMapsUseCase`?